### PR TITLE
corrections-v1

### PR DESCRIPTION
**Pull request by Fractal-Tess**
Hey there,

I noticed some issues on the docs, so here is a pr.

Please take these changes with a huge grain of salt, as I'm not a native speaker and pretty much all of the changes that are into this pr come from internal instinct. Please double-check them in case I outright butchered something.

Also, I haven't yet looked into the js API, so perhaps I'll make a pr for that in the future!

Cheers!

### CODE REVIEW

File: docs/guides/building/app-size.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ But keep in mind: **The best dependency is no dependency**, meaning that you sho
 
 ### Optimize Images
 
-According to the [Http Archive], images are the [biggest contributor to website weight][http archive report, image bytes]. So if your app includes have background images or icons, make sure to optimize them!
+According to the [Http Archive], images are the [biggest contributor to website weight][http archive report, image bytes]. So if your app includes images, make sure to optimize them!
```
Comment (amrbashir) [inline docs/guides/building/app-size.md]:
```suggestion
According to the [Http Archive], images are the [biggest contributor to website weight][http archive report, image bytes]. So if your app includes images or icons, make sure to optimize them!
```

File: docs/guides/distribution/sign-linux.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ You can embed a signature in the AppImage by setting the following environment v
 
 - **SIGN**: set to `1` to sign the AppImage.
 - **SIGN_KEY**: optional variable to use a specific GPG Key ID for signing.
-- **APPIMAGETOOL_SIGN_PASSPHRASE**: the signing key password. If unset, gpg shows a dialog so you can input it, so you must set this when running in automated tasks.
+- **APPIMAGETOOL_SIGN_PASSPHRASE**: the signing key password. If unset, gpg shows a dialog so you can input it. So you must set this when running automated tasks.
```
Comment (amrbashir) [inline docs/guides/distribution/sign-linux.md]: why `. So` instead of `, so`?

File: docs/guides/distribution/sign-macos.md [hunk 3]
```diff
@@ -154,7 +154,7 @@ The workflow pulls the secrets from GitHub and defines them as environment varia
 
 [tauri-apps/tauri#592]: https://github.com/tauri-apps/tauri/issues/592
 [apple developer program]: https://developer.apple.com/programs/
-[notarizing macOS software before distribution]: https://developer.apple.com/documentation/security/notarizing_macos_software_before_distribution
```
Comment (amrbashir) [inline docs/guides/distribution/sign-macos.md]: `macOS` is the correct spelling of the OS.

File: docs/guides/features/system-tray.md [hunk 4]
```diff
@@ -163,9 +163,9 @@ fn main() {
     .system_tray(SystemTray::new().with_menu(tray_menu))
     .on_system_tray_event(|app, event| match event {
       SystemTrayEvent::MenuItemClick { id, .. } => {
-        // get a handle to the clicked menu item
+        // get a handle on the clicked menu item
```
Comment (amrbashir) [inline docs/guides/features/system-tray.md]:
```suggestion
        // get a handle to the clicked menu item
```

The handle points to the clicked menu item

File: docs/guides/getting-started/prerequisites.md [hunk 5]
```diff
@@ -310,7 +311,7 @@ If you don't see this information, your Rust installation might be broken. Pleas
...
 [download webview2]: https://developer.microsoft.com/en-us/microsoft-edge/webview2/#download-section
 [rustup.sh]: https://sh.rustup.rs
-[Nix Flakes]: https://nixos.wiki/wiki/Flakes
-[direnv's Flakes integration]: https://nixos.wiki/wiki/Flakes#Direnv_integration
-[Nix Shell]: https://nixos.wiki/wiki/Development_environment_with_nix-shell
-[direnv's Shell integration]: https://nixos.wiki/wiki/Development_environment_with_nix-shell#direnv
+[nix flakes]: https://nixos.wiki/wiki/Flakes
+[direnv's flakes integration]: https://nixos.wiki/wiki/Flakes#Direnv_integration
+[nix shell]: https://nixos.wiki/wiki/Development_environment_with_nix-shell
+[direnv's shell integration]: https://nixos.wiki/wiki/Development_environment_with_nix-shell#direnv
```
Comment (amrbashir) [inline docs/guides/getting-started/prerequisites.md]: why change these? these are just links

File: docs/references/architecture/inter-process-communication/isolation.md [hunk 6]
```diff
@@ -53,7 +53,7 @@ Because the point of the Isolation application is to protect against Development
 
 In this example, we will make a small hello-world style Isolation application and hook it up to an imaginary existing Tauri application. It will do no verification of the messages passing through it, only print the contents to the WebView console.
 
-For the purposes of this example, let's imagine we are in the same directory as `tauri.conf.json`. The existing Tauri application has it's `distDir` set to `../dist`.
+For this example, let's imagine we are in the same directory as `tauri.conf.json`. The existing Tauri application has it's `distDir` set to `../dist`.
```
Comment (amrbashir) [inline docs/references/architecture/inter-process-communication/isolation.md]: I like the original wording, is there a reason to change it?

File: docs/references/architecture/readme.md [hunk 7]
```diff
@@ -108,7 +108,7 @@ The Tauri-Apps organisation maintains two "upstream" crates from Tauri, namely T
 
 ### [TAO](https://github.com/tauri-apps/tao)
 
-Cross-platform application window creation library in Rust that supports all major platforms like Windows, macOS, Linux, iOS and Android. Written in Rust, it is a fork of [winit](https://github.com/rust-windowing/winit) that we have extended for our own needs like menu bar and system tray.
```
Comment (amrbashir) [inline docs/references/architecture/readme.md:111]: Can we keep this bit of information?

**Review by amrbashir — CHANGES_REQUESTED** [review]
Thank you for your efforts.

File: docs/guides/distribution/sign-linux.md (hunk 2 above)
Comment (Fractal-Tess) [reply docs/guides/distribution/sign-linux.md]: At first glance, it seemed like it was most appropriate to divide the sentence into two parts, as there is a repeating `so`.
That being said, I'm unsure if doing so causes this description to lose meaning in the context of the documented item.

File: docs/guides/distribution/sign-macos.md (hunk 3 above)
Comment (Fractal-Tess) [reply docs/guides/distribution/sign-macos.md]: Yeah, that is correct! 
This seems to be an error on my part.
My guess is that it got auto-formated when I copy-pasted the text

File: docs/guides/features/system-tray.md (hunk 4 above)
Comment (Fractal-Tess) [reply docs/guides/features/system-tray.md]: I'm unable to confirm which variant is correct. 
I looked into the definition of prepositions, but their use in the current context (programming) might differ slightly from what the norm is. 
I believe that either of them works in the currently abstract context.

File: docs/guides/getting-started/prerequisites.md (hunk 5 above)
Comment (Fractal-Tess) [reply docs/guides/getting-started/prerequisites.md]: Copy-paste ide formatting.
It wasn't my intention to change these.
In future prs, I'll be more careful about such things happening.

File: docs/references/architecture/inter-process-communication/isolation.md (hunk 6 above)
Comment (Fractal-Tess) [reply docs/references/architecture/inter-process-communication/isolation.md]: While I skipped most of the text in the docs as it is not feasibly possible for me to go over it in a single night, I've tried to reduce some sentences by removing needlessly emphasised text and still capturing the original meaning.
If you, however, feel like emphasis here and it is good, then I wholeheartedly welcome the idea!

File: docs/references/architecture/readme.md (hunk 7 above)
Comment (Fractal-Tess) [reply docs/references/architecture/readme.md:111]: Well, the purpose of that sentence is to explain the purpose of cloning and extending `winit`. 
The addition of `like menu bar and system tray` is information that reinforces the main point of the sentence, which was already established.
However, as I previously mentioned in a comment, this emphasis could be left in if so desired.

File: docs/guides/distribution/sign-linux.md (hunk 2 above)
Comment (amrbashir) [reply docs/guides/distribution/sign-linux.md]: Maybe remove `So` from the second sentence.

File: docs/guides/getting-started/prerequisites.md (hunk 5 above)
Comment (amrbashir) [reply docs/guides/getting-started/prerequisites.md]: No problem, just revert back.

File: docs/guides/distribution/sign-macos.md (hunk 3 above)
Comment (amrbashir) [reply docs/guides/distribution/sign-macos.md]: I see, just revert back then.

File: docs/references/architecture/inter-process-communication/isolation.md (hunk 6 above)
Comment (amrbashir) [reply docs/references/architecture/inter-process-communication/isolation.md]: Yeah, I think the extra emphasis here is good.

File: docs/references/architecture/readme.md (hunk 7 above)
Comment (amrbashir) [reply docs/references/architecture/readme.md:111]: Yeah, I think the extra details here is needed.

**Review by amrbashir — APPROVED** [review]
Thank you

**Review by FabianLars — APPROVED** [review]
Don't worry about the lower case links btw. This is the primary styling we use in the repo and most formatters do that so it will probably be lower case in one of the next PRs again 🤷

**Pull request merged**